usaspending_api/conftest_helpers.py

```diff
@@ -63,15 +63,12 @@ def _add_contents(self):
     @staticmethod
     def _refresh_materialized_views():
         """
-        These two materialized views are used by transaction_delta_view.sql, so
-        we will need to refresh them in order for transaction_delta_view to see
-        changes to their underlying tables.
+        This materialized view is used by transaction_delta_view.sql, so
+        we will need to refresh it in order for transaction_delta_view to see
+        changes to the underlying tables.
         """
         with connection.cursor() as cursor:
-            cursor.execute(
-                "refresh materialized view universal_award_matview; "
-                "refresh materialized view universal_transaction_matview;"
-            )
+            cursor.execute("refresh materialized view universal_transaction_matview;")
```

Comment:
why is awards gone?

Reply:
This entire PR revolves around removing `universal_award_matview`

Reply:
...

Reply:
right